packages/@react-spectrum/s2/src/NumberField.tsx

```diff
@@ -51,7 +51,11 @@ export interface NumberFieldProps extends
    *
    * @default 'M'
    */
-  size?: 'S' | 'M' | 'L' | 'XL'
+  size?: 'S' | 'M' | 'L' | 'XL',
+  /**
+   * Temporary text that occupies the text input when it is empty.
```

Comment:
inherit from `InputProps`? hopefully then it comes withe the MDN description and link?